Jenkinsfile

```diff
@@ -33,6 +33,21 @@ node {
     }
 }
 
+// Parse and handle the result of Kola
+boolean checkKolaSuccess(currentBuild) {
```

Comment:
Yeah, this is the kind of thing that we can push up into [coreos-ci-lib](https://github.com/coreos/coreos-ci-lib) at some point.